components/Footer/Footer.js

```diff
@@ -3,74 +3,116 @@ import Link from 'next/link';
-        </a>
-
-        <SocialMedia />
-      </div>
-
-      <div className={classNames(styles.footerGrouping, styles.logoGrouping)}>
-        <img src="/static/images/icons/operation_code_logo.svg" alt="Operation Code Logo" />
+      <div className={classNames(styles.footerWrapper, styles.row)}>
+        <div className={classNames(styles.footerGrouping, styles.socialGrouping)}>
+          <div className={classNames(styles.logoGrouping)}>
+            <img src="/static/images/operation_code_logo.png" alt="Operation Code Logo" />
+          </div>
+          <div className={classNames(styles.capitalize, styles.marginBottom)}>Connect With Us!</div>
+          <SocialMedia />
+        </div>
 
-        <p className={styles.copyright}>
-          Copyright {`${new Date().getUTCFullYear()}`}
-          <br />
-          Operation Code™
-        </p>
-      </div>
+        <div className={classNames(styles.footerGrouping, styles.linksGrouping, styles.capitalize)}>
+          <div className={styles.linksRow}>
+            <div className={styles.linksColumn}>
+              <Link href="/about">
+                <a>About Us</a>
+              </Link>
+              <Link href="/team">
+                <a>Team</a>
+              </Link>
+              <Link href="/contact">
+                <a>Contact Us</a>
+              </Link>
+              <Link href="/faq">
+                <a>FAQ</a>
+              </Link>
+            </div>
+            <div className={styles.linksColumn}>
+              <Link href="/who_we_serve">
+                <a>Who We Serve</a>
+              </Link>
+              <Link href="/code_schools">
+                <a>Code Schools</a>
+              </Link>
+              <Link href="/job_board">
+                <a>Job Board</a>
+              </Link>
+              <Link href="/events">
+                <a>Events</a>
+              </Link>
+              <Link href="/blog">
+                <a>Blog</a>
+              </Link>
+            </div>
+          </div>
+          <div className={styles.linksRow}>
+            <div className={styles.linksColumn}>
+              <Link href="/get_involved">
+                <a>Get Involved</a>
+              </Link>
+              <Link href="/become_a_mentor">
+                <a>Become a Mentor</a>
+              </Link>
+              <Link href="/support">
+                <a>Support OC</a>
+              </Link>
+              <Link href="/donate">
+                <a>Donate</a>
+              </Link>
+              <Link href="/feedback">
+                <a>Feedback</a>
+              </Link>
+            </div>
+            <div className={styles.linksColumn}>
+              <Link href="/resources">
+                <a>Resources</a>
+              </Link>
+              <Link href="/press">
+                <a>Press</a>
+              </Link>
+              <Link href="/branding">
+                <a>Branding</a>
+              </Link>
+              <Link href="/sitemap">
```

Comment:
@dmarchante is this necessary to have within the footer?

Reply:
Ima go ahead and say no... Sitemap is relevant for SEO purposes, but we have no need to route traffic to an XML mapping.

Please remove 👍